frontend/src/metabase/parameters/components/FilterApplyButton/FilterApplyButton.styled.ts

```diff
@@ -0,0 +1,40 @@
+import styled from "@emotion/styled";
+
+import { space } from "metabase/styled-components/theme";
+import Button from "metabase/core/components/Button/Button";
+
+export const TRANSITION_CLASSNAMES_PREFIX = "filter-apply-button";
+export const BUTTON_TRANSITION_DURATION = 200;
+
+export const ApplyButton = styled(Button)<{ isVisible: boolean }>`
+  margin-left: auto;
+  margin-top: ${space(1)};
```

Comment:
Previously I use `align-items: baseline` on the parent container, but that causes a slight shift when adding a filter. So I use fixed margin to make the apply button stay still.
![apply button shift](https://user-images.githubusercontent.com/1937582/232624252-262d857c-f47f-48fb-99fd-99608f6435d7.gif)